res/values/strings_hi.arb

```diff
@@ -313,12 +313,17 @@
   "incorrect_seed": "दर्ज किया गया पाठ मान्य नहीं है।",
   "introducing_cake_pay": "परिचय Cake Pay!",
   "invalid_input": "अमान्य निवेश",
+  "invoice": "चालान",
+  "invoice_created": "चालान बनाया गया",
   "invoice_details": "चालान विवरण",
   "is_percentage": "है",
   "last_30_days": "पिछले 30 दिन",
   "learn_more": "और अधिक जानें",
   "light_theme": "रोशनी",
-  "loading_your_wallet": "अपना बटुआ लोड कर रहा है",
```

Comment:
why was this removed?

Reply:
I had to do a bunch of manual .arb file editing to get the lightning_invoice strings to translate correctly, I must've just missed this in the process
this is one of the languages that randomly translated some of my substitution words incorrectly